public/plugin.tsx

```diff
@@ -169,6 +169,7 @@ export class ObservabilityPlugin
   constructor(initializerContext: PluginInitializerContext) {
     this.config = initializerContext.config.get<PublicConfig>();
   }
+  private featureFlagStatus: boolean = false;
```

Comment:
can this variable naming be more explicit? looking at `featureFlagStatus` it's hard to tell what the features is..

Reply:
Yes I will change that.

Reply:
Since the backport is failing, I will manually change it in 2.x first and backport. I will push another commit for change it on main.

Reply:
I have updated here https://github.com/opensearch-project/dashboards-observability/pull/2144.